flink-staging/flink-streaming/flink-streaming-core/src/main/java/org/apache/flink/streaming/api/operators/AbstractUdfStreamOperator.java

```diff
@@ -84,6 +84,10 @@ public void confirmCheckpointCompleted(long checkpointId, long timestamp) throws
 		}
 	}
 
+	public F getUserFunction() {
```

Comment:
Added a public getter, thus the StreamGraph creation from APIs is more testable.